mmdet/models/detectors/yolact.py

```diff
@@ -25,12 +25,40 @@ def __init__(self,
         self.segm_head = build_head(segm_head)
         self.mask_head = build_head(mask_head)
 
-    def forward_dummy(self, img):
+    def forward_dummy(self, imgs):
         """Used for computing network flops.
 
         See `mmdetection/tools/analysis_tools/get_flops.py`
         """
-        raise NotImplementedError
+        # img_metas do not influence the flops calculation
+        dummy_img_metas = [
```

Comment:
We only need to consider network forward instead of simple_test. You can refer to two-stage:  https://github.com/open-mmlab/mmdetection/blob/master/mmdet/models/detectors/two_stage.py#L72

Reply:
I'll fix that. Is there any requirements for the outputs? Do they need to be real results of the forward? Can I simply generate the outputs?

Reply:
No need for real output, as long as flop can be calculated reasonably.

Reply:
fixed